packages/salesforcedx-vscode-apex-replay-debugger/src/utils/index.ts

```diff
@@ -4,9 +4,5 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import { getRootWorkspacePath } from '@salesforce/salesforcedx-utils-vscode/out/src/workspaces';
-import * as path from 'path';
 
-export function getLogDirPath(): string {
```

Comment:
Moved this to the utils module so we can use it in apex and apex replay debugger extensions